internal/sleepable/sleepable_timer_test.go

```diff
@@ -175,15 +175,16 @@ func TestTimerEdgeCases(t *testing.T) {
 
 	t.Run("very long duration", func(t *testing.T) {
 		start := clock.Now()
-		target := start.Add(24 * time.Hour)
+		target := start.Add(100 * time.Millisecond) // Use a shorter duration for testing
```

Comment:
The subtest name no longer describes the case being tested with the new parameters.